tests/containers/bulkDownload/archive/AwardDataArchiveContainer-test.jsx

```diff
@@ -57,23 +57,16 @@ describe('AwardDataArchiveContainer', () => {
             const container = mount(<AwardDataArchiveContainer/>);
 
             const expectedFilterState = {
-                agency: {
-                    id: 'all',
-                    name: 'All'
-                },
-                type: {
-                    name: 'mockType',
-                    display: 'Mock Type'
-                },
-                fy: '2019'
```

Comment:
Was this failing because of the fiscal year changeover? If so, we should mock the date instead of removing `fy`.

Reply:
@ebdabbs it appeared to me that we weren't actually testing the `fy` value in this context. The test itself seems to be checking the `type` property.

Is that incorrect?

I'm afraid it might be non-trivial to go from using a live date in this test to mocking the date.